scripts/services/getConfig.ts

```diff
@@ -0,0 +1,145 @@
+import { compat, types as T } from "../deps.ts";
+
+export const getConfig: T.ExpectedExports.getConfig = compat.getConfig({
+  "nodes": {
+      "type": "list",
+      "subtype": "union",
+      "name": "Lightning Nodes",
+      "description": "List of Lightning Network node instances to manage",
+      "range": "[1,*)",
+      "default": [
+        "lnd"
+      ],
+      "spec": {
+        "display-as": "{{name}}",
+        "unique-by": "name",
+        "name": "Node Implementation",
+        "tag": {
+            "id": "type",
+            "name": "Type",
+            "description": "- LND: Lightning Network Daemon from Lightning Labs\n- CLN: Core Lightning from Blockstream\n",
+            "variant-names": {
+              "lnd": "Lightning Network Daemon (LND)",
+              "c-lightning": "Core Lightning (CLN)"
+            }
+        },
+        "default": "lnd",
+        "variants": {
+            "lnd": {
+              "name": {
+                  "type": "string" as const,
+                  "name": "Node Name",
+                  "description": "Name of this node in the list",
+                  "default": "Embassy LND",
+                  "nullable": false
+              },
+              "connection-settings": {
+                  "type": "union" as const,
+                  "name": "Connection Settings",
+                  "description": "The Lightning Network Daemon node to connect to.",
+                  "tag": {
+                    "id": "type",
+                    "name": "Type",
+                    "description": "- Internal: The Lightning Network Daemon service installed to your Embassy.\n- External: A Lightning Network Daemon instance running on a remote device (advanced).\n",
+                    "variant-names": {
+                        "internal": "Internal",
+                        "external": "External"
+                    }
```

Comment:
Thought we were deprecating external nodes

Reply:
No, LND and CLN can be external since they use https for the connection